core/src/main/java/org/bitcoinj/core/CheckpointManager.java

```diff
@@ -66,19 +66,18 @@
  * different concept of checkpoints that are used to hard-code the validity of blocks that violate BIP30 (duplicate
  * coinbase transactions). Those "checkpoints" can be found in NetworkParameters.</p>
  *
- * <p>The file format consists of the string "CHECKPOINTS 1", followed by a uint32 containing the number of signatures
+ * <p>The file format is textual, one value per line.
+ * It consists of the magic string "TXT CHECKPOINTS 1", followed by the number of signatures
  * to read. The value may not be larger than 256 (so it could have been a byte but isn't for historical reasons).
  * If the number of signatures is larger than zero, each 65 byte ECDSA secp256k1 signature then follows. The signatures
  * sign the hash of all bytes that follow the last signature.</p>
  *
- * <p>After the signatures come an int32 containing the number of checkpoints in the file. Then each checkpoint follows
- * one after the other. A checkpoint is 12 bytes for the total work done field, 4 bytes for the height, 80 bytes
- * for the block header and then 1 zero byte at the end (i.e. number of transactions in the block: always zero).</p>
+ * <p>After the signatures come the number of checkpoints in the file. Then each checkpoint follows
+ * one per line in compact format as a base64-encoded blob.</p>
```

Comment:
Their should probably be a link to the `StoredBlock` V2 methods here.

Reply:
Added link to the non-V2 method (the next PR will change this to V2).